src/mixins/filing-template-mixin.ts

```diff
@@ -832,4 +936,66 @@ export default class FilingTemplateMixin extends DateMixin {
       })
     }
   }
+
+  /**
```

Comment:
I moved these here from the Legal Services class since they don't belong there but they are what THIS mixin is all about.